openwisp-utils-qa-checks

```diff
@@ -93,17 +96,32 @@ runcheckcommit() {
   fi
 }
 
+runmakemigrationcheck() {
+  echo "Running Make Migrations Check"
```

Comment:
None of the tests are doing the "Running" echo, can you please remove it from this one as well in favor of consistency!